### ISSUE TITLE
Javascript file mission:  src="js/dt-codetoweb.dataTables.editor.js"></script>

### ISSUE THREAD

**Issue by lablushah** [CodeToWeb/Laravel-DatatablesEditor#2]
Thank you for this wonderful package; however, I didn't find the subject js file. I would appreciate your providing this mission js file. So far my implementation works fine except I could not create new record. It gives me status 500 error. On submitting (clicking Create button), it gives 500. Edit, Delete and Search are working fine.

**Comment by ekandreas** [comment]
Isn't the [editor.js-file](https://editor.datatables.net/purchase/index) licensed?

**Comment by lablushah** [comment]
Ya, I should be licensed. I tried with their trial version. However, I gave up and started using Vue.js. Its awesome: easily understandable and manageable. Anyway, thanks for reply.

**Comment by ekandreas** [comment]
Yes, Vue is awesome but a completely different tech, purpose and solution. Autogenerating crud forms with vue? I would be happy to try it :-)
